benchmarks/asv.conf.json

```diff
@@ -35,7 +35,6 @@
 	    "pytest": [],
 	    "nose": [],
 	    "psutil": [],
-	    "mock": [],
```

Comment:
going to revert this change based on @orbeckst's comment: https://github.com/MDAnalysis/mdanalysis/pull/2778#discussion_r443195483